tests/unit_tests/test_suassystem_sda_converter.py

```diff
@@ -1,10 +1,10 @@
+import unittest
 from SUASSystem import *
 from SDA import *
-import unittest
 import numpy
 import interop
 
-class SDAIntegrationTestCase(unittest.TestCase):
+class TestSDAConverter(unittest.TestCase):
```

Comment:
I forgot to previously mention this naming convention. The names of a test class should be ```ClassToTestWithTestCase```. So in this case, it should be ```SDAConverterTestCase```. If possible, could you look through all of the other unit test files to make sure they are named this way?